labdao/plex
# privy login on web app

## Summary

Replacement for #682. Users login to web app using Privy. [**Demo.**](https://www.loom.com/share/0d28d521ae0a4fc08da43b201a630aee?sid=8b664af6-9b83-43e3-8fd4-24e7c0bb28e6)

## Steps to Test

1. `cd frontend`
2. `npm install`
3. Get Privy app ID and set `NEXT_PUBLIC_PRIVY_APP_ID` to its value
4. Clear DB
5. Boot up frontend and backend DB
6. Login with two separate accounts (MetaMask and email or GitHub)
7. Confirm wallet address are separate
8. For email or GitHub login, confirm you can export private key
9. Upload datafile and tool
10. Create flow
11. Confirm flow works `bacalhau describe <bacalhau-job-id-in-flow-cid>`
12. Logout
